packages/amazon_security_lake/data_stream/event/elasticsearch/ingest_pipeline/default.yml

```
@@ -0,0 +1,1234 @@
+           }
+         }
+  - convert:
+      field: ocsf.severity_id
+      tag: convert_severity_id_to_long
+      if: ctx.ocsf?.severity_id != ''
+      type: long
+      ignore_missing: true
+      on_failure:
+        - remove:
+            field: ocsf.severity_id
+            ignore_missing: true
+        - append:
+            field: error.message
+            value: 'Processor {{{_ingest.on_failure_processor_type}}} with tag {{{_ingest.on_failure_processor_tag}}} in pipeline {{{_ingest.on_failure_pipeline}}} failed with message: {{{_ingest.on_failure_message}}}'
+  - set:
+      field: event.severity
+      tag: set_event_severity
+      copy_from: ocsf.severity_id
+      ignore_empty_value: true
+  - date:
+      field: ocsf.start_time_dt
+      tag: date_start_time_dt
+      target_field: ocsf.start_time_dt
+      formats:
+        - ISO8601
+      if: ctx.ocsf?.start_time_dt != null && ctx.ocsf.start_time_dt != ''
+      on_failure:
+        - remove:
+            field: ocsf.start_time_dt
+            ignore_missing: true
+        - append:
+            field: error.message
+            value: 'Processor {{{_ingest.on_failure_processor_type}}} with tag {{{_ingest.on_failure_processor_tag}}} in pipeline {{{_ingest.on_failure_pipeline}}} failed with message: {{{_ingest.on_failure_message}}}'
+  - set:
+      field: event.start
+      tag: set_event_start_from_start_time_dt
+      copy_from: ocsf.start_time_dt
+      ignore_empty_value: true
+  - date:
+      field: ocsf.start_time
+      tag: date_start_time
+      target_field: ocsf.start_time
+      formats:
+        - UNIX_MS
+      if: ctx.ocsf?.start_time != null && ctx.ocsf.start_time != ''
+      on_failure:
+        - remove:
+            field: ocsf.start_time
+            ignore_missing: true
+        - append:
+            field: error.message
+            value: 'Processor {{{_ingest.on_failure_processor_type}}} with tag {{{_ingest.on_failure_processor_tag}}} in pipeline {{{_ingest.on_failure_pipeline}}} failed with message: {{{_ingest.on_failure_message}}}'
+  - set:
+      field: event.start
+      tag: set_event_start_from_start_time
+      copy_from: ocsf.start_time
+      ignore_empty_value: true
+  - convert:
+      field: ocsf.status_id
+      tag: convert_status_id_to_string
+      if: ctx.ocsf?.status_id != ''
+      type: string
+      ignore_missing: true
+      on_failure:
+        - remove:
+            field: ocsf.status_id
+            ignore_missing: true
+        - append:
+            field: error.message
+            value: 'Processor {{{_ingest.on_failure_processor_type}}} with tag {{{_ingest.on_failure_processor_tag}}} in pipeline {{{_ingest.on_failure_pipeline}}} failed with message: {{{_ingest.on_failure_message}}}'
+  - set:
+      field: event.outcome
+      value: unknown
+      if: ctx.ocsf.status_id != null && ctx.ocsf.status_id == '0'
```

```suggestion
      if: ctx.ocsf?.status_id != null && ctx.ocsf.status_id == '0'
```
Same for below 2 processors